### DISCUSSION TITLE
Showcases: I need you!

### DISCUSSION THREAD

**Discussion by estruyf** [estruyf/vscode-front-matter#84] [Announcements]
On the new [frontmatter.codes](https://frontmatter.codes) website, there is a showcase section. It would be cool if you could add your website to it and show the world you are supporting this extension/CMS.

https://frontmatter.codes/showcase

In order to add your website, all you have to do is create an issue: [showcase issue](https://github.com/estruyf/vscode-front-matter/issues/new?assignees=&labels=&template=showcase.md&title=Showcase:)

![Screenshot 2021-09-01 at 09 47 44](https://user-images.githubusercontent.com/2900833/131633039-737ffdca-f35f-488d-8cf2-93ba6673a27a.png)

**Comment by Fasani** [comment]
Where would we add?

I blogged about this software a bit here:
https://www.michaelfasani.com/2020/plugins-markdown-editing-experience-vs-code/

Maybe it is outdated now, maybe some of my plugins are no longer needed?

**Reply by estruyf** [reply]
The extension has been having significant improvements and features over the last couple of weeks. Where it was a tool to help you manage your articles, it is more and more becoming a real CMS in VS Code. 

There are indeed a couple of extensions that you would not need any longer, but you can keep on using what you like to use of course 🤓.
